common/trace.go

```diff
@@ -0,0 +1,15 @@
+package common
+
+import (
+	"context"
+
+	"go.opencensus.io/trace"
+)
+
+func StartSpan(ctx context.Context, name string, o ...trace.StartOption) (context.Context, *trace.Span) {
```

Comment:
Consider how this would be if we would add Datadog as well

Reply:
Is this supposed to be used by middleware or by the servers?

Reply:
The middlewares use this to initiate their span.

https://github.com/SKF/go-enlight-middleware/pull/1/files#diff-8ed40bf10e6744ba950aa3a5f7e0285332334d3ffe55a0ad1d8fbb3adb2dbd53R57